inference-engine/src/low_precision_transformations/src/network_helper.cpp

```diff
@@ -875,7 +875,7 @@ size_t CNNNetworkHelper::getInputChannelsCount(const CNNLayer& layer) {
         return insertData->getDims()[1];
     }
     default: {
-        THROW_IE_EXCEPTION << "Not supported layout " << insertData->getLayout();
+        return insertData->getDims()[1];
```

Comment:
Can insertData->getDims().size() be 1 (insertData->getLayout() == Layout::C)? Does it mean that this line fails?